Articles/Packages/Extending/creating-a-package.md

```diff
@@ -0,0 +1,141 @@
+# Creating a Package
+
+The goal of this tutorial will be to take something that extends Umbraco and create it as a package. The tutorials starting point will be creating a package out of the dashboard from the [Creating a Custom Dashboard tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md). The process will be the same for most packages so feel free to follow along with something else.
+
+## Creating a package in the backoffice
+
+If you have something you want to make into a package, you can do so through the Umbraco backoffice. 
+
+First go to the `Packages` section and select `Created` in the top right corner. Next, choose the `Create package` button.
+
+![Buttons to select for creating a package in the backoffice](images/creating-package-menu.png)
+
+On the `Create package` page there are 4 sections with different info to fill out. Some of the info is mandatory, most of it is not. First of all you should give the package a name at the top - we will call our dashboard the same as in the [tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md): `Custom Welcome Dashboard`.
+
+### The Package Properties section
+
+This section contains mostly meta data about the package and the creator. We will fill in the following things:
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Url | https://umbraco.com | This url will be shown as the "packages url" when others install it. Will mostly be a Github repo or similar |
+| Version | 1.0.0 | This is automatically set to 1.0.0 but can be changed or increased manually if needed |
+| Icon Url | _Blank_ | Not a mandatory value, but will appear as the package icon different places in the backoffice if set |
+| Umbraco version | 9.0.0 | It will automatically select the Umbraco version you are currently using. This is then set as the compatible version for the package |
+| Author | Your name | Here you get to take credit for your awesome work! |
+| Author URL | Your website or maybe a link to your Twitter profile | Will link to this from the author name certain places in the backoffice |
+| Contributors | _Blank_ | Here you can add the names of other contributors if you have any |
+| License | MIT License | Will be set to MIT by default. Please consider how you want your package licensed. If in doubt when deciding an open-source license there are [good resources available](https://choosealicense.com/licenses/). |
+| License URL | http://opensource.org/licenses/MIT | Will be set to the URL for the MIT license, can be changed as needed |
+| Package readme | This will add a dashboard to your content section. <br> <br>The dashboard will show each user the most recent nodes they have saved. | This will be shown when someone looks at the package in the package dashboard |
```

Comment:
```suggestion
| Package readme | This will add a dashboard to your content section. <br> <br>The dashboard will show each user the most recent nodes they have saved. | This will be shown when someone looks at the package in the packages dashboard. |
```